lntest/harness.go

```diff
@@ -573,6 +576,47 @@ func (n *NetworkHarness) StopNode(node *HarnessNode) error {
 	return node.stop()
 }
 
+//SaveProfilesPages hits profiles pages of all active nodes and writes it to disk
+//using a similar naming scheme as to the regular set of logs.
+func (n *NetworkHarness) SaveProfilesPages() {
+	for _, node := range n.activeNodes {
+		url := "http://localhost:%d/debug/pprof/goroutine?debug=1"
+
+		resp, err := http.Get(fmt.Sprintf(url, node.TestPort))
+
+		if err != nil {
+			fmt.Printf("Failed: Node id is %d; Node name is %s. Exited with error: \n"+
```

Comment:
Useful! 👍 Maybe "Failed to get profile page"?

Reply:
Done.